### PR TITLE
fix the message_type of client

### PR DESCRIPTION
**Pull request by 1C4nfaN**
Based on the pr  #190 
Added a oneway test case.

### CODE REVIEW

File: tests/oneway.thrift [hunk 1]
```diff
@@ -0,0 +1,4 @@
+service echo
+{
```
Comment (aisk) [inline tests/oneway.thrift:2]:
```suggestion
service echo {
```

File: tests/test_oneway.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+class Dispatcher(object):
+    def Test(self, req):
+        print("Get req msg: %s" % req)
+
+        assert req == "Hello!"
+
+
+oneway_thrift = thriftpy2.load("oneway.thrift", module_name="oneway_thrift")
+multiprocessing.set_start_method("fork")
```
Comment (aisk) [inline tests/test_oneway.py:15]: This have side effect globally in the lifetime of running all the tests, can we avoid it?

Comment (1C4nfaN) [reply tests/test_oneway.py:15]: You are right, I try to solve this

**Comment by 1C4nfaN** [comment]
hi @aisk , Could you please take a look at this pr again?  Thanks～

**Review by aisk — APPROVED** [review]

**Comment by 1C4nfaN** [comment]
Thanks for Aisk~ 
hi @ethe , Could you please check at this PR? Thanks～

**Pull request merged**